lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -82,7 +84,13 @@ def __init__(self, *args, **kwargs):
         self.tbl_num = self._service_manager.get_table_num(self.APP_NAME)
         self.next_table = self._service_manager.get_table_num(INGRESS)
         self.loop = kwargs['loop']
-        self._uplink_port = OFPP_LOCAL
+        self.uplink_port_name = kwargs['config']['ovs_uplink_port_name']
+        if self.config.enable_nat is False and self.uplink_port_name is not None:
+            self._uplink_port = BridgeTools.get_ofport(
+                self.uplink_port_name,
+            )
+        else:
+            self._uplink_port = OFPP_LOCAL
```

Comment:
This PR is trying to implement two different functionality.
1. Non-NAT
2. Add support for IPv6.

Please separate it in two separate PRs with its own unit tests.

Reply:
@pshelar , we have fixed it as per your suggestion.
1) Created PR Support for Non-NAT mode in 5G-SA #13288  and merged into master.
2) Rebased with master and added UT for ipv6 solicitation
python/magma/pipelined/tests/test_ipv6_solicitation_5g.py

We have verified with UERANSIM for below functionality and along with end-to-end traffic flows:
Router Solicitation
Router Advertisement
IPv6 traffic with TCP.

Attached logs and pcap file for same.

[tc2.txt](https://github.com/magma/magma/files/9132581/tc2.txt)
[IPv6_Traffic_flows.zip](https://github.com/magma/magma/files/9132655/IPv6_Traffic_flows.zip)